project-reports/2024/2024-Q2-Hyperledger-Indy.md

```diff
@@ -0,0 +1,167 @@
+- [indy-sdk] -- Deprecated
+
+[indy-vdr]: https://github.com/hyperledger/indy-vdr
+[indy-shared-rs]: https://github.com/hyperledger/indy-shared-rs
+[indy-cli-rs]: https://github.com/hyperledger/indy-cli-rs
+[indy-sdk]: https://github.com/hyperledger/indy-sdk
+[Indy SDK]: https://github.com/hyperledger/indy-sdk
+[Aries Askar]: https://github.com/hyperledger/aries-askar
+[Hyperledger AnonCreds]: https://github.com/hyperledger/anoncreds-rs
+
+### **Specifications**
+
+- [did:indy Specification], [did:indy Specification source repository]
+- [did:indy Networks]
+- [Indy HIPE] (Hyperledger Indy Project Enhancements)
+- [indy-did-method]
+
+[did:indy Specification]: https://hyperledger.github.io/indy-did-method/
+[did:indy Specification source repository]: https://github.com/hyperledger/indy-did-method
+[did:indy Networks]: https://github.com/hyperledger/indy-did-networks
+[Indy HIPE]: https://github.com/hyperledger/indy-hipe
+[indy-did-method]: https://github.com/hyperledger/indy-did-method
+
+## Project Health
+
+The project continued to have steady contributor activity. Per the [Indy
+Quarterly Activity Dashboard], in the months January through March, 2024 there
+were 145 commits (about the same as the previous quarterly report quarterly, and
+up from the previous quarter) from 34 contributors (up significantly from the
+previous quarterly report quarter, but down about 20% from last quarter).
+
+[Indy Quarterly Activity Dashboard]: https://insights.lfx.linuxfoundation.org/foundation/hyp/overview?project=indy&repository=&dateFilters=Last%20Quarter&dateRange=2024-01-01%20to%202024-03-31&compare=PP&granularity=week&hideBots=true&routedFrom=Github
+
+## Questions/Issues for the TOC
+
+From the previous quarterly report:
+
+> See the idea in that came out of the Second Hyperledger Indy Ecosystem Summit:
+>
+> Is there a way for the Hyperledger Foundation to enable a group to collect
+funds for an open source objective, and the use those funds to enable a code
+contribution?
+
+This was answered in the review of that last quarterly report. The best option
+available is through the Linux Foundations [LFX Crowdfunding Platform]. Some
+work is being done in the Besu community towards such a process, but the
+administrative and legal overhead is high, beyond what is appropriate for Indy.
+
+[LFX Crowdfunding Platform]: https://lfx.linuxfoundation.org/tools/crowdfunding/
+
+### Issues from previous reports
+
+#### **Diversity of Contributor Community**
+
+See the updated type of information in the appropriate section of this report.
+
+## Releases
+
+Release since the last quarterly report -- those after September 30, 2023:
+
+- indy-node - v1.13.2
+- indy-plenum - v1.13.1
+- indy-vdr -  - v0.4.1
+- indy-node-container - None
+- indy-shared-rs - v1.1.0, v1.1.1
+
+## Overall Activity in the Past Quarter
+
+A key focus in this past quarter has been on
+completing the official release of Indy supporting Ubuntu 20.24, including the
```

Comment:
```suggestion
completing the official release of Indy supporting Ubuntu 24.04, including the
```

Reply:
I assume you mean Ubuntu 24.04.
I'm not an expert in this area, but I believe we've found that binaries built on older Ubuntu versions will work on newer Ubuntu versions. However binaries built on newer Ubuntu versions will not always work on older Ubuntu versions. For this reason Fabric tends to keep CI including release jobs that build binaries on the older Ubuntu LTS releases that are still in standard maintenance (currently 20.04).
Since you brought up the topic of Ubuntu versions, I'm just wondering if anybody knows whether this is a reasonable practice in general.

Reply:
I did mean 20.24, unfortunately.  The need to update the code starts with the version of Python supported on the different Ubuntu releases and leads to the various dependencies that change with each OS release. Each needs a full update/test/release cycle.